src/utils.js

```diff
@@ -616,6 +616,69 @@ export const snakeCaseToCamelCase = (snakeCaseStr) => {
         .join('');
 };
 
+/**
+ * Scrolls to the bottom of a page, or until it times out.
+ * @param {Object} page
+ * @param {Number} [maxTimeout=0] How many seconds to scroll for. If 0, will scroll until bottom of page.
+ * @param {Number} [waitForDynamicContent=4] How many seconds to wait for no new content to load before exit.
+ * @returns {Promise}
+ * @memberOf utils
+ */
+export const infiniteScroll = async function (page, maxTimeout = 0, waitForDynamicContent = 4) {
```

Comment:
Please use options object for these settings, it makes the code much more readable, and it let's you extend to code in the future

Reply:
BTW, values like `maxTimeout` should always contain the unit, otherwise, a rocket will crash (https://en.wikipedia.org/wiki/Mars_Climate_Orbiter#Cause_of_failure). Call it `timeoutSecs`, no need for "max"...